packages/fsapp/src/beta-app/router/router.web.tsx

```diff
@@ -43,10 +43,9 @@ export class FSRouter extends FSRouterBase {
   private constructScreen = (
     route: Route,
     loading: boolean,
-    routeDetails: ActivatedRoute,
-    prefix?: string
+    routeDetails: ActivatedRoute
   ): JSX.Element | JSX.Element[] => {
```

Comment:
We can't get rid of `prefix` if we still want to support ParentRoutes alongside RouteCollections